week6.3-ex1.py

```diff
@@ -0,0 +1,14 @@
+from typing import List
+
+
+def words_length(sentence: str) -> List[int]:
+    """
+    The function receives a sentence and returns the length of each word in it, in order.
```

Comment:
What do you mean by "in order"?

Reply:
In the order of the words in the sentence.